templates/parachain/runtime/src/lib.rs

```diff
@@ -756,12 +756,16 @@ impl_runtime_apis! {
 	}
 
 	impl sp_genesis_builder::GenesisBuilder<Block> for Runtime {
-		fn create_default_config() -> Vec<u8> {
-			create_default_config::<RuntimeGenesisConfig>()
+		fn build_state(config: Vec<u8>) -> sp_genesis_builder::Result {
+			build_state::<RuntimeGenesisConfig>(config)
 		}
 
-		fn build_config(config: Vec<u8>) -> sp_genesis_builder::Result {
-			build_config::<RuntimeGenesisConfig>(config)
+		fn get_preset(id: &Option<sp_genesis_builder::PresetId>) -> Option<Vec<u8>> {
```

Comment:
Given that these are templates usable in testing and education, it would be a good idea to add presets to them. Do you intend in doing this? if not I can open a branch on top of this PR.

Reply:
So my intention here was to get to consensus about the general approach. Once agreed upon and merged I planned to migrate all the runtimes one by one (including fellowship runtimes).

I'll appreciate help with this.